src/main/angband-initializer.cpp

```diff
@@ -197,10 +197,9 @@ static void init_angband_aux(const std::string &why)
  */
 static void put_title(void)
 {
-    char title[120];
-    put_version(title);
+    const auto title = get_version();
 
-    int col = (80 - strlen(title)) / 2;
+    int col = (80 - title.length()) / 2;
```

Comment:
'auto' is desirable.

Reply:
The latest forced push changes that declaration to auto.  In line with other recommendations, it makes some other changes to the prototype of put_title() and another declaration there.